pages/docs/tooling/tooling-aste.md

```diff
@@ -69,16 +91,17 @@ For demo usage please check [ASTE Tutorial](https://github.com/precice/tutorials
 
 ### preciceMap
 
-The main tool used for mapping. It is a wrapper around preCICE interface. It can be run in serial or parallel. Mapped data can be a vector or scalar.
+The main tool used for mapping. It is a wrapper around the preCICE interface. It can be run in serial or parallel. There are two modes available, namely, mapper mode and replay mode. The first mode is used to assess mapping data from one mesh to another; later, one is mainly developed for diagnostic and adaptor development purposes. In Replay mode, ASTE replicates one participant of preCICE.
```

Comment:
```suggestion
The main tool used for mapping. It is a wrapper around the preCICE interface. It can be run in serial or parallel. There are two modes available, namely, mapper mode and replay mode. The first mode is used to assess mapping data from one mesh to another; later, one is mainly developed for diagnostic and adapter development purposes. In Replay mode, ASTE replicates one participant of preCICE.
```

Reply:
Thank you.